### PR TITLE
save 'level' reference in project properties, link to project edit url

### PR DESCRIPTION
**Pull request by laurelfan**
- save level reference in project properties
- link to project edit url from project list
- fix delete from project list

### CODE REVIEW

File: shared/js/initApp.js [hunk 1]
```diff
@@ -286,8 +287,18 @@ if (appOptions.droplet) {
...
     .then(loadSource(appOptions.locale + '/blockly_locale'));
   if (appOptions.level.isProjectLevel) {
+    // example paths:
+    // edit: /p/artist#7uscayNy-OEfVERwJg0xqQ==/edit
+    // view: /p/artist#7uscayNy-OEfVERwJg0xqQ==
     var app_id = location.hash.slice(1);
     if (app_id) {
+      // TODO ugh, we should use a router. maybe we should use angular :p
+      var params = app_id.split("/")
+      if (params.length > 1 && params[1] == "edit") {
```
Comment (joshlory) [inline shared/js/initApp.js]: Could do something like `/\/edit$/.test(location.hash)` but what you have is fine.  Also I don't think we want to call it `app_id` yet on line 294.  Maybe a temp var until we can split out the real `app_id`?

**Comment by joshlory** [comment]
:+1: LGTM

**Comment by joshlory** [comment]
Whoops, one nit: shouldn't saving the project take me to the /#id/edit URL?

**Comment by laurelfan** [comment]
yeah... probably should implement the url and editUrl methods on that project 'object' too...

**Pull request merged**